syntax = "proto3";

package config;


message config{
    uint64 commit_handle = 1;
    bool enable_compress = 2;
    bool enable_encrypt = 3;
    bool enable_meta = 4;
    bool verbose = 5;
    optional bytes magic_code = 6;
}
